README.md

```diff
@@ -0,0 +1,54 @@
+# Project Template
+
+## Overview
+
+### EVT-core
+For an overview of EVT please see the following links:
+
+[EVT-core Documentation](https://evt-core.readthedocs.io/)
+
+[EVT-core Github](https://github.com/RIT-EVT/EVT-core/)
+
+### Template Project
+
+This project-template serves as the skeleton template that is used for
+all EVT board repositories.  It contains the following capabilities:
+
+- EVT-core is included as a submodule and compiled as a library for
+the board
+- Template is set up for a library containing code pertinent to the board 
+to be built.
+- The template supports an arbitrary number of executable targets to be built
+and deployed onto a microcontroller
+  - These contain targets to be run on a specific EVT custom boards
+  - Also contain utilities for debugging and debugging
+- Framework for auto-generated and built documentation using Sphinx and
+hosted on `readthedocs.io`
+
+## Steps to Setup a New Project
+
+1) Create a new repo based on the project-template
+   1) From [project-template](https://github.com/RIT-EVT/project-template) click `Use this template`
+   2) Choose a repo name and make sure it's public for auto-documentation to work
+   3) If the owner is not set to RIT-EVT then you need to update the .gitmodules file URL
+2) Clone the new repo
+   1) `git clone <URL>`
+3) Update the EVT submodule
+   1) `git submodule update --init --recusive && git pull`
+   2) The above command can be used at any point to update the submodule from remote
+4) Import the project into Read the Docs
+   1) Log in to [readthedocs.io](https://readthedocs.org/)
+   2) Navigate to dashboard page
+   3) Import the Repository.  Follow instructions on webpage
```

Comment:
Do we want the owner of the readthedocs.io to be the team lead, or just whichever member is in charge of the specific project?  I see this being a problem once members start graduating.

Reply:
Right now there is an "EVT" account handling the read the docs hosting. We can add "maintainers" to projects, so members would be "maintainers" and the projects would be owned by "EVT"

Reply:
@cbolles Would you mind updating the README accordingly?  I'm not sure how we set up the hosting with the maintainer accounts.